diff --git a/node_modules/three-globe/dist/three-globe.mjs b/node_modules/three-globe/dist/three-globe.mjs
index cc53c4e..ad1df28 100644
--- a/node_modules/three-globe/dist/three-globe.mjs
+++ b/node_modules/three-globe/dist/three-globe.mjs
@@ -1899,7 +1899,7 @@ var computeGeoKde = /*#__PURE__*/function () {
             });
           });
           computeNode = computeShaderFn().compute(vertexGeoCoords.length);
-          renderer = new WebGPURenderer();
+          renderer = new WebGPURenderer({ forceWebGL: true });
           _context.n = 2;
           return renderer.computeAsync(computeNode);
         case 2:
